tests/test_api.py

```diff
@@ -1474,6 +1476,7 @@ def test_split_query(self):
                                 self.base_time + timedelta(minutes=self.minutes)
                             ).isoformat(),
                             "orderby": "-timestamp",
+                            "granularity": 60,
```

Comment:
Why were these added.

Reply:
I just realized that I broke things with my change. I will fix and revert this.